blocks/library/table/table-block.js

```diff
@@ -89,6 +89,7 @@ export default class TableBlock extends wp.element.Component {
 						<li>
 							<DropdownMenu
 								icon="editor-table"
+								label={ wp.i18n.__( 'Edit Table' ) }
```

Comment:
>I didn't want to change also all the other ones, because outside of the scope of this PR.

Sure, especially in smaller pull requests like this one, I'm fine with some general refactoring sprinkled in as well. In an ideal world we'd have separate individual pull requests to refactor, but alas the world is not ideal and I find such PRs are few and far between.

But agree with the sentiment generally 👍